Mathlib/Tactic/NormNum/Result.lean

```diff
@@ -279,6 +281,12 @@ and `q` is the value of `n / d`. -/
     ∀ (inst : Q(DivisionRing $α) := by assumption) (q : Rat) (n : Q(ℤ)) (d : Q(ℕ))
       (proof : Q(IsRat $x $n $d)), Result x := Result'.isRat
 
+/-- The result is `proof : isRat x n d`, where `n` is either `.ofNat lit` or `.negOfNat lit`
+with `lit` a raw nat literal and `d` is a raw nat literal (not 0 or 1),
+and `q` is the value of `n / d`. -/
```

Comment:
This doc comment is a verbatim copy of the `Result.isRat` comment. Did you mean to have something different here?